backend/conduit/organizations/views.py

```diff
@@ -101,17 +101,13 @@ def unfollow_an_organization(slug):
     return organization
 
 
-# # Add Member to organization
-# @blueprint.route('api/organizations/<slug>/members', methods=('POST',))
-# @jwt_required
-# @marshal_with(organization_schema)
-# def add_member(slug):
-#     # search user by username
-#     # get user_id
-#     user = User.query.filter_by(user_id=user_id).first()
-#     organization = Organization.query.filter_by(slug=slug).first()
-#     if not organization:
-#         raise InvalidUsage.organization_not_found()
-#     organization.add_member(user)
-# # Remove Member
-# # Get all Members & Mods
\ No newline at end of file
+# Get all Members & Mods
+@blueprint.route('/api/organizations/<slug>/members', methods=('GET',))
```

Comment:
This route returns all the data from the whole organization. It should only return the members and moderators. Perhaps make another schema that returns the moderators and members only inside of an organization. Should look like:
```
{
    "organization": {
        "members": [],
        "moderators": []
     } 
}
```